javafx-archetype-simple/src/main/resources/archetype-resources/pom.xml

```diff
@@ -36,4 +38,23 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <profile>
+            <id>openjfx</id>
+            <activation>
+                <jdk>[9,)</jdk>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>org.openjfx</groupId>
+                    <artifactId>javafx-controls</artifactId>
+#if ("8" == ${javafx-version})
```

Comment:
This means that if user's JDK is 9+, JavaFX will be set to 11 regardless the `javafx-version` being set to 8 (1.8), doesn't it?